README.md

```diff
@@ -1,2 +1,8 @@
 # GitHubAiAPI
 This is Example to test AI plugin for automatic code review
+
+# Resources
+https://docs.github.com/en/rest/quickstart?apiVersion=2022-11-28
+
+# Owner
+Saurabh Chauhan
```

Comment:
The 'Owner' section could include more information, such as contact details or a brief bio.